Error building VideoInfoScanner.cpp

<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
## Bug report
### Describe the bug
Here is a clear and concise description of what the problem is:
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
<!--- A bug report that is not clear will be closed -->
<!--- Put your text below this line -->
Kodi build fails on CoreELEC.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:
<!--- Tell us what should happen -->
<!--- Put your text below this line -->
To pass build.

## Actual Behavior
<!--- Tell us what happens instead -->
<!--- Put your text below this line -->
```
[883/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoInfoScanner.cpp.o
FAILED: [code=1] build/video/CMakeFiles/video.dir/VideoInfoScanner.cpp.o 
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/bin/aarch64-libreelec-linux-gnu-g++ -DFFMPEG_VER_SHA=7.0.0 -DFMT_SHARED -DHAS_AIRPLAY -DHAS_AIRTUNES -DHAS_ALSA -DHAS_AVAHI -DHAS_DBUS -DHAS_EGL -DHAS_FILESYSTEM_NFS -DHAS_FILESYSTEM_SMB -DHAS_GLES=3 -DHAS_ISO9660PP -DHAS_LIBAMCODEC=1 -DHAS_LIRC -DHAS_MARIADB -DHAS_NEON -DHAS_PULSEAUDIO -DHAS_PYTHON -DHAS_UDFREAD -DHAS_WEB_INTERFACE -DHAS_WEB_SERVER -DHAS_ZEROCONF -DHAVE_DRM_MODIFIER_NAME -DHAVE_HDR_OUTPUT_METADATA -DHAVE_LIBBLUETOOTH -DHAVE_LIBBLURAY -DHAVE_LIBBLURAY_BDJ -DHAVE_LIBCAP -DHAVE_LIBCEC -DHAVE_LIBDOVI -DHAVE_LIBUDEV -DHAVE_LIBXSLT -DHAVE_NEW_CROSSGUID -DPCRE2_STATIC -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DSPDLOG_SHARED_LIB -DTAGLIB_STATIC -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132 -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/lib -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/xbmc -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/xbmc/platform/linux -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/xbmc/cores/VideoPlayer -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/.aarch64-libreelec-linux-gnu/build -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/.aarch64-libreelec-linux-gnu/build/include -I/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/xbmc/platform/posix -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/harfbuzz -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/glib-2.0 -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/lib/glib-2.0/include -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fribidi -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/freetype2 -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/.aarch64-libreelec-linux-gnu/build/cores/RetroPlayer/messages -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/lzo -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/taglib -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/amcodec -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/libdrm -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/libxml2 -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/libcec -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/p8-platform -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/dbus-1.0 -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/lib/dbus-1.0/include -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/python3.13 -isystem /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/samba-4.0 -march=armv8-a+crc+fp+simd -mabi=lp64 -Wno-psabi -mtune=cortex-a53 -mno-outline-atomics -Wall -pipe  -O3 -fomit-frame-pointer -DNDEBUG -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -DDUMMYOPT -std=c++20 -flto=12 -fno-fat-lto-objects -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_GETMNTENT_R=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_UPNP=1 -DHAS_OPTICAL_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/lib/kodi\" -DINSTALL_PATH=\"/usr/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -MD -MT build/video/CMakeFiles/video.dir/VideoInfoScanner.cpp.o -MF build/video/CMakeFiles/video.dir/VideoInfoScanner.cpp.o.d -o build/video/CMakeFiles/video.dir/VideoInfoScanner.cpp.o -c /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/build/kodi-b75e7966e413dd1712983ca95fa1ace53d7cf132/xbmc/video/VideoInfoScanner.cpp
In file included from /CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/format.h:41,
                 from ../xbmc/addons/Scraper.h:22,
                 from ../xbmc/video/VideoDatabase.h:13,
                 from ../xbmc/video/VideoInfoScanner.h:12,
                 from ../xbmc/video/VideoInfoScanner.cpp:9:
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/base.h: In instantiation of 'static void fmt::v11::detail::value<Context>::format_custom(void*, fmt::v11::parse_context<typename Context::char_type>&, Context&) [with T = ADDON::ContentType; Formatter = fmt::v11::formatter<ADDON::ContentType>; Context = fmt::v11::context; typename Context::char_type = char]':
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/base.h:2255:19:   required from 'constexpr fmt::v11::detail::value<Context>::value(T&, fmt::v11::detail::custom_tag) [with T = ADDON::ContentType; typename std::enable_if<has_formatter<T, typename Context::char_type>(), int>::type <anonymous> = 0; Context = fmt::v11::context]'
 2255 |     custom.format = format_custom<value_type, formatter<value_type, char_type>>;
      |     ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/base.h:2235:65:   required from 'constexpr fmt::v11::detail::value<Context>::value(T&) [with T = ADDON::ContentType; typename std::enable_if<(std::integral_constant<bool, (((((((std::is_class<T>::value || std::is_enum<T>::value) || std::is_union<T>::value) || std::is_array<_Up>::value) && (! fmt::v11::detail::has_to_string_view<T, void>::value)) && (! fmt::v11::detail::is_named_arg<T>::value)) && (! fmt::v11::detail::use_format_as<T>::value)) && (! fmt::v11::detail::use_format_as_member<typename std::remove_const<_Tp>::type, std::integral_constant<bool, true> >::value))>::value || (!1)), int>::type <anonymous> = 0; Context = fmt::v11::context]'
 2235 |   FMT_CONSTEXPR20 FMT_INLINE value(T& x) : value(x, custom_tag()) {}
      |                                                                 ^
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/base.h:2808:20:   required from 'constexpr fmt::v11::detail::format_arg_store<Context, NUM_ARGS, NUM_NAMED_ARGS, DESC> fmt::v11::make_format_args(T& ...) [with Context = context; T = {ADDON::ContentType, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; int NUM_ARGS = 2; int NUM_NAMED_ARGS = 0; long long unsigned int DESC = 223]'
 2808 |   return {{args...}};
      |                    ^
../xbmc/utils/log.h:106:76:   required from 'static void CLog::Log(spdlog::level::level_enum, fmt::v11::format_string<T ...>, Args&& ...) [with Args = {ADDON::ContentType, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; fmt::v11::format_string<T ...> = fmt::v11::fstring<ADDON::ContentType, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
  106 |     GetInstance().FormatAndLogInternal(level, format, fmt::make_format_args(args...));
      |                                                       ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~
../xbmc/utils/log.h:89:8:   required from 'static void CLog::Log(int, fmt::v11::format_string<T ...>, Args&& ...) [with Args = {ADDON::ContentType, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; fmt::v11::format_string<T ...> = fmt::v11::fstring<ADDON::ContentType, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
   89 |     Log(MapLogLevel(level), format, std::forward<Args>(args)...);
      |     ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../xbmc/video/VideoInfoScanner.cpp:627:18:   required from here
  627 |         CLog::Log(LOGERROR, "VideoInfoScanner: Unknown content type {} ({})", info2->Content(),
      |         ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  628 |                   CURL::GetRedacted(pItem->GetPath()));
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/CoreELEC/build.CoreELEC-Amlogic-no.aarch64-22/toolchain/aarch64-libreelec-linux-gnu/sysroot/usr/include/fmt/base.h:2275:29: error: passing 'const fmt::v11::formatter<ADDON::ContentType>' as 'this' argument discards qualifiers [-fpermissive]
 2275 |     ctx.advance_to(cf.format(*static_cast<qualified_type*>(arg), ctx));
      |                    ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../xbmc/addons/Scraper.h:202:18: note:   in call to 'constexpr auto fmt::v11::formatter<ADDON::ContentType>::format(const ADDON::ContentType&, FormatContext&) [with FormatContext = fmt::v11::context]'
  202 |   constexpr auto format(const ADDON::ContentType& type, FormatContext& ctx)
      |                  ^~~~~~
[884/1378] Building CXX object build/video/CMakeFiles/video.dir/ViewModeSettings.cpp.o
[885/1378] Building CXX object build/video/dialogs/CMakeFiles/video_dialogs.dir/GUIDialogFullScreenInfo.cpp.o
[886/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoInfoTag.cpp.o
[887/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoLibraryQueue.cpp.o
[888/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoItemArtworkHandler.cpp.o
[889/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoUtils.cpp.o
[890/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoThumbLoader.cpp.o
[891/1378] Building CXX object build/video/dialogs/CMakeFiles/video_dialogs.dir/GUIDialogAudioSettings.cpp.o
[892/1378] Building CXX object build/video/dialogs/CMakeFiles/video_dialogs.dir/GUIDialogSubtitles.cpp.o
[893/1378] Building CXX object build/video/dialogs/CMakeFiles/video_dialogs.dir/GUIDialogSubtitleSettings.cpp.o
[894/1378] Building CXX object build/video/CMakeFiles/video.dir/VideoDatabase.cpp.o
ninja: build stopped: subcommand failed.
```
## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
<!--- Put your text below this line -->
CLog::Log(LOGERROR, "VideoInfoScanner: Unknown content type {} ({})", info2->Content(),
                  CURL::GetRedacted(pItem->GetPath()));
must be
        CLog::Log(LOGERROR, "VideoInfoScanner: Unknown content type {} ({})", TranslateContent(info2->Content()),
                  CURL::GetRedacted(pItem->GetPath()));


### To Reproduce
Steps to reproduce the behavior:
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
<!--- Put your text below this line -->
1. 
2.
3.


### Debuglog
<!--- Put your text below this line -->
<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
The debuglog can be found here:



### Screenshots 
Here are some links or screenshots to help explain the problem:
<!--- Put your text below this line -->



## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- Put your text below this line -->



### Your Environment
Used Operating system:
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- Put your text below this line. Checkboxes can easily be ticked once issue is created -->
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version:



<!--- End of this issue -->
*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

--- a/xbmc/addons/Scraper.cpp
+++ b/xbmc/addons/Scraper.cpp
@@ -936,7 +936,7 @@ std::vector<CScraperUrl> CScraper::FindMovie(XFILE::CCurlFile &fcurl,
 
   CLog::LogF(LOGDEBUG,
              "Searching for '{}' using {} scraper (path: '{}', content: '{}', version: '{}')",
-             sTitle, Name(), Path(), ADDON::TranslateContent(Content()), Version().asString());
+             sTitle, Name(), Path(), Content(), Version().asString());
 
   std::vector<CScraperUrl> vcscurl;
   if (IsNoop())
@@ -1077,8 +1077,7 @@ std::vector<CMusicAlbumInfo> CScraper::FindAlbum(CCurlFile &fcurl,
 {
   CLog::LogF(LOGDEBUG,
              "Searching for '{} - {}' using {} scraper (path: '{}', content: '{}', version: '{}')",
-             sArtist, sAlbum, Name(), Path(), ADDON::TranslateContent(Content()),
-             Version().asString());
+             sArtist, sAlbum, Name(), Path(), Content(), Version().asString());
 
   std::vector<CMusicAlbumInfo> vcali;
   if (IsNoop())
@@ -1177,7 +1176,7 @@ std::vector<CMusicArtistInfo> CScraper::FindArtist(CCurlFile &fcurl, const std::
 {
   CLog::LogF(LOGDEBUG,
              "Searching for '{}' using {} scraper (file: '{}', content: '{}', version: '{}')",
-             sArtist, Name(), Path(), ADDON::TranslateContent(Content()), Version().asString());
+             sArtist, Name(), Path(), Content(), Version().asString());
 
   std::vector<CMusicArtistInfo> vcari;
   if (IsNoop())
@@ -1265,8 +1264,7 @@ VIDEO::EPISODELIST CScraper::GetEpisodeList(XFILE::CCurlFile& fcurl, const CScra
     return vcep;
 
   CLog::LogF(LOGDEBUG, "Searching '{}' using {} scraper (file: '{}', content: '{}', version: '{}')",
-             scurl.GetFirstThumbUrl(), Name(), Path(), ADDON::TranslateContent(Content()),
-             Version().asString());
+             scurl.GetFirstThumbUrl(), Name(), Path(), Content(), Version().asString());
 
   if (m_isPython)
   {
@@ -1365,7 +1363,7 @@ bool CScraper::GetVideoDetails(XFILE::CCurlFile& fcurl,
   CLog::LogF(LOGDEBUG,
              "Reading {} '{}' using {} scraper (file: '{}', content: '{}', version: '{}')",
              fMovie ? MediaTypeMovie : MediaTypeEpisode, scurl.GetFirstThumbUrl(), Name(), Path(),
-             ADDON::TranslateContent(Content()), Version().asString());
+             Content(), Version().asString());
 
   video.Reset();
 
@@ -1409,8 +1407,7 @@ bool CScraper::GetVideoDetails(XFILE::CCurlFile& fcurl,
 bool CScraper::GetAlbumDetails(CCurlFile &fcurl, const CScraperUrl &scurl, CAlbum &album)
 {
   CLog::LogF(LOGDEBUG, "Reading '{}' using {} scraper (file: '{}', content: '{}', version: '{}')",
-             scurl.GetFirstThumbUrl(), Name(), Path(), ADDON::TranslateContent(Content()),
-             Version().asString());
+             scurl.GetFirstThumbUrl(), Name(), Path(), Content(), Version().asString());
 
   if (m_isPython)
     return PythonDetails(ID(), "url", scurl.GetFirstThumbUrl(),
@@ -1446,8 +1443,7 @@ bool CScraper::GetArtistDetails(CCurlFile &fcurl,
 
   CLog::LogF(LOGDEBUG,
              "Reading '{}' ('{}') using {} scraper (file: '{}', content: '{}', version: '{}')",
-             scurl.GetFirstThumbUrl(), sSearch, Name(), Path(), ADDON::TranslateContent(Content()),
-             Version().asString());
+             scurl.GetFirstThumbUrl(), sSearch, Name(), Path(), Content(), Version().asString());
 
   if (m_isPython)
     return PythonDetails(ID(), "url", scurl.GetFirstThumbUrl(),
@@ -1484,8 +1480,7 @@ bool CScraper::GetArtwork(XFILE::CCurlFile &fcurl, CVideoInfoTag &details)
 
   CLog::LogF(LOGDEBUG,
              "Reading artwork for '{}' using {} scraper (file: '{}', content: '{}', version: '{}')",
-             details.GetUniqueID(), Name(), Path(), ADDON::TranslateContent(Content()),
-             Version().asString());
+             details.GetUniqueID(), Name(), Path(), Content(), Version().asString());
 
   if (m_isPython)
     return PythonDetails(ID(), "id", details.GetUniqueID(),

--- a/xbmc/addons/Scraper.h
+++ b/xbmc/addons/Scraper.h
@@ -199,7 +199,7 @@ template<>
 struct fmt::formatter<ADDON::ContentType> : fmt::formatter<std::string_view>
 {
   template<typename FormatContext>
-  constexpr auto format(const ADDON::ContentType& type, FormatContext& ctx)
+  constexpr auto format(const ADDON::ContentType& type, FormatContext& ctx) const
   {
     return fmt::formatter<std::string_view>::format(enumToSV(type), ctx);
   }

--- a/xbmc/video/VideoInfoScanner.cpp
+++ b/xbmc/video/VideoInfoScanner.cpp
@@ -405,7 +405,7 @@ CVideoInfoScanner::~CVideoInfoScanner()
       CLog::Log(
           LOGINFO,
           "VideoInfoScanner: Plugin '{}' does not support media library scanning for '{}' content",
-          CURL::GetRedacted(strDirectory), TranslateContent(content));
+          CURL::GetRedacted(strDirectory), content);
       return true;
     }
 
@@ -815,7 +815,7 @@ CVideoInfoScanner::~CVideoInfoScanner()
       return retVal < 0 ? InfoRet::CANCELLED : InfoRet::NOT_FOUND;
 
     CLog::Log(LOGDEBUG, "VideoInfoScanner: Fetching url '{}' using {} scraper (content: '{}')",
-              url.GetFirstThumbUrl(), info2->Name(), TranslateContent(info2->Content()));
+              url.GetFirstThumbUrl(), info2->Name(), info2->Content());
     const std::unordered_map<std::string, std::string> uniqueIDs{{identifierType, identifier}};
 
     if (GetDetails(pItem, {}, url, info2,
@@ -912,7 +912,7 @@ CVideoInfoScanner::~CVideoInfoScanner()
       return retVal < 0 ? InfoRet::CANCELLED : InfoRet::NOT_FOUND;
 
     CLog::Log(LOGDEBUG, "VideoInfoScanner: Fetching url '{}' using {} scraper (content: '{}')",
-              url.GetFirstThumbUrl(), info2->Name(), TranslateContent(info2->Content()));
+              url.GetFirstThumbUrl(), info2->Name(), info2->Content());
 
     if (GetDetails(pItem, {}, url, info2,
                    (result == InfoType::COMBINED || result == InfoType::OVERRIDE) ? loader.get()
@@ -1002,7 +1002,7 @@ CVideoInfoScanner::~CVideoInfoScanner()
       return retVal < 0 ? InfoRet::CANCELLED : InfoRet::NOT_FOUND;
 
     CLog::Log(LOGDEBUG, "VideoInfoScanner: Fetching url '{}' using {} scraper (content: '{}')",
-              url.GetFirstThumbUrl(), info2->Name(), TranslateContent(info2->Content()));
+              url.GetFirstThumbUrl(), info2->Name(), info2->Content());
 
     if (GetDetails(pItem, {}, url, info2,
                    (result == InfoType::COMBINED || result == InfoType::OVERRIDE) ? loader.get()
@@ -1655,7 +1655,8 @@ CVideoInfoScanner::~CVideoInfoScanner()
       }
     }
 
-    CLog::Log(LOGDEBUG, "VideoInfoScanner: Adding new item to {}:{}", TranslateContent(content), CURL::GetRedacted(pItem->GetPath()));
+    CLog::Log(LOGDEBUG, "VideoInfoScanner: Adding new item to {}:{}", content,
+              CURL::GetRedacted(pItem->GetPath()));
     long lResult = -1;
 
     if (content == ContentType::MOVIES)

